Makefile

```diff
@@ -168,6 +168,10 @@ CONCORD_BFT_ADDITIONAL_RUN_PARAMS+=\
 	--network host
 endif
 
+CONCORD_BFT_ADDITIONAL_CTEST_RUN_PARAMS="--output-on-failure"
+FAILED_CASES_FILE=/concord-bft/build/apollogs/latest/failed_cases.txt
```

Comment:
please document the new tests output files in the PR

Reply:
In the PR description, sorry

Reply:
Done